rest/payloads/generated/actions/structs.conjure.go

```diff
@@ -1371,8 +1365,20 @@ type TransactionPayload struct {
 	// Optional because it is added later so some payloads don't have it.
 	Value *string `json:"value" conjure-docs:"Optional because it is added later so some payloads don't have it."`
 	// Optional because it is added later so some payloads don't have it.
-	Nonce   *string `json:"nonce" conjure-docs:"Optional because it is added later so some payloads don't have it."`
-	AlertId *string `json:"alertId"`
+	Nonce *string `json:"nonce" conjure-docs:"Optional because it is added later so some payloads don't have it."`
+	// Optional because it is added later so some payloads don't have it.
+	Gas *string `json:"gas" conjure-docs:"Optional because it is added later so some payloads don't have it."`
+	// Optional because it is added later so some payloads don't have it.
+	GasUsed *string `json:"gasUsed" conjure-docs:"Optional because it is added later so some payloads don't have it."`
+	// Optional because it is added later so some payloads don't have it.
+	CumulativeGasUsed *string `json:"cumulativeGasUsed" conjure-docs:"Optional because it is added later so some payloads don't have it."`
+	// Optional because it is added later so some payloads don't have it.
+	GasPrice *string `json:"gasPrice" conjure-docs:"Optional because it is added later so some payloads don't have it."`
+	// Optional because it is added later so some payloads don't have it.
+	GasTipCap *string `json:"gasTipCap" conjure-docs:"Optional because it is added later so some payloads don't have it."`
+	// Optional because it is added later so some payloads don't have it.
+	GasFeeCap *string `json:"gasFeeCap" conjure-docs:"Optional because it is added later so some payloads don't have it."`
```

Comment:
Should this be in the same PR?

Reply:
Those fields are generated from conjure earlier, but not copied to tenderly-cli at that moment as they are not used here, so they ended up in this PR with other changes in the generated model. Do you think I should remove them from this PR?